mobile-browser-based-version/server/README.md

```diff
@@ -18,8 +18,19 @@ Tasks are stored in ```tasks.json```. The models are declared in ```models.js```
 
 ### Running the server
```

Comment:
better call it helper server (only keeping list of available peers per task, not data)